wlinux-setup.d/shell-opts.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+source "/usr/local/wlinux-setup.d/common.sh"
+
+function InputRc {
+WLINUX_STRING="### WLINUX OPTIMISED DEFAULTS"
+
+echo "Installing optimised inputrc commands to /etc/inputrc"
+
+if [[ ! -f "/etc/inputrc" ]] ; then
+	sudo touch /etc/inputrc
+fi
+
+cat /etc/inputrc | while read line
+do
+	if [[ $line == *WLINUX_STRING ]] ; then
+		echo "Previous wlinux inputrc customizations detected. Cancelling install..."
+		whiptail --title "Warning!" --msgbox "Previous install of wlinux inputrc customizations detected. To reinstall, please edit \"/etc/inputrc\" with your favourite text editor and remove all of the text between (and including) the lines $WLINUX_STRING" 15 95
+		return
+	fi
+done
+
+# Write Carlos' custom inputrc to global inputrc (see: https://github.com/crramirez/shellprefs/blob/master/.inputrc)
+echo ${WLINUX_STRING}								| sudo tee -a /etc/inputrc
```

Comment:
i recommend using `    cat << 'EOF' >> xxx
yyy
EOF` notation to avoid over work

Reply:
Ah thanks for the heads up, will modify this :+1:

Reply:
Also worth mentioning I still need to implement your changes you suggested regarding sourcing of common.sh, just letting you know it's not forgotten!